test/Smidge.Tests/SmidgeHelperTests.cs

```diff
@@ -1,29 +1,30 @@
 using System;
-using Microsoft.AspNetCore.Hosting;
 using Microsoft.AspNetCore.Http;
 using Moq;
 using Smidge.CompositeFiles;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using Dazinator.Extensions.FileProviders;
+using Dazinator.Extensions.FileProviders.InMemory;
+using Dazinator.Extensions.FileProviders.InMemory.Directory;
 using Microsoft.Extensions.FileProviders;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
-using Smidge;
 using Smidge.Cache;
 using Smidge.Hashing;
 using Smidge.FileProcessors;
 using Smidge.Options;
+using Smidge.Tests.Helpers;
 using Xunit;
 
 namespace Smidge.Tests
 {
     public class SmidgeHelperTests
     {
-        private readonly IUrlManager _urlManager = Mock.Of<IUrlManager>();
+        private readonly IUrlManager _urlManager;// = Mock.Of<IUrlManager>();
```

Comment:
can we remove this commented out code?

Reply:
Have removed the commented out code, and made a couple of other tidy ups.